airflow/decorators/__init__.pyi

```diff
@@ -52,10 +52,10 @@ class TaskDecoratorCollection:
     def python(
         self,
         *,
-        multiple_outputs: Optional[bool] = None,
```

Comment:
This will not work without "from future" i believe (unless you are on Python 3.9). This is .pyi file so maybe it behaves differently ?

Reply:
I am not sure how to validate this. I am running the test suite with 3.8, and the pre-commit hook are also run with 3.8, static checks were fine locally, but I don't have enough knowledge to assert that this suffice in validating the stub file

Reply:
It's only valid for MyPy and autocompletion. MyPy runs using Python 3.7 docker image actually, so if it works there, it means that it works and MyPy does not pay attention to "from_future" in the stub files. If you can actually check it in Python 3.7 IDE to see if autocomplete works for one of those, we should be good :)

Reply:
answered in the other thread (my bad) https://github.com/apache/airflow/pull/26872#discussion_r987410569, seems to work fine.